### PR TITLE
CONCF-830 | add mercury cct01

### PR DESCRIPTION
**Pull request by tomasznapieralski**
Check functionality:
* check if url is on /
* check visibility of:
* * title
* * stats
* * feature content
* * curated content
* * trending articles
* * trending videos
* check correct elements order

### CODE REVIEW

**Comment by tomasznapieralski** [comment]
@justnpT @bognix

File: src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/CuratedPageObject.java [hunk 1]
```diff
@@ -1,15 +1,129 @@
...
+  private WebElement mobilePrefooter;
 
   public CuratedPageObject(WebDriver driver) {
     super(driver);
   }
+
+  public boolean isMobileTopLeaderboardVisible() {
+    try {
+      waitForElementVisibleByElement(mobileTopLeaderboard, 5, 1000);
+    } catch (TimeoutException e) {
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/pageobjectsfactory/pageobject/mercury/CuratedPageObject.java]: move 5 and 1000 to variables. Its used in so many places

File: src/test/java/com/wikia/webdriver/testcases/mercurytests/CuratedMainPageTests.java [hunk 2]
```diff
@@ -20,13 +21,57 @@ public void prepareTest() {
...
     CuratedPageObject cc = new CuratedPageObject(driver);
-  }
+    PageObjectLogging
+        .log("Current URL", "is set on " + ROOT_PATH, "is not set on " + ROOT_PATH,
+             cc.isUrlPathEqualTo(ROOT_PATH));
+    PageObjectLogging
+        .log("AD Mobile Top Leaderboard", "is visible", "is not visible",
+             cc.isMobileTopLeaderboardVisible());
+    PageObjectLogging
+        .log("Article Title", "is visible", "is not visible", cc.isArticleTitleVisible());
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/testcases/mercurytests/CuratedMainPageTests.java]: move "is visibile" and "is not visible" messages to variables

File: src/test/java/com/wikia/webdriver/common/contentpatterns/MercuryArticles.java [hunk 3]
```diff
@@ -20,9 +17,18 @@
   public static final String QUESTION_MARK = "Question?mark?question";
   public static final String COLON = "Colon:colon:colon";
 
-  //Articles prepared for mercurycc.wikia.com wiki
+  //Articles prepared for mercurycc.wikia.com wiki (with FC, CC, TA and TV)
```
Comment (justnpT) [inline src/test/java/com/wikia/webdriver/common/contentpatterns/MercuryArticles.java]: let's not udse short cuts for that

**Pull request merged**